src/pages/TCY/components/TCYNavIndicator.tsx

```diff
@@ -0,0 +1,53 @@
+import type { ResponsiveValue } from '@chakra-ui/react'
+import { Box, Center } from '@chakra-ui/react'
+import type { Property } from 'csstype'
+import { motion } from 'framer-motion'
+
+import { useTCYClaims } from '../queries/useTcyClaims'
+
+const position: ResponsiveValue<Property.Position> = { base: 'absolute', '2xl': 'relative' }
+const right = { base: -1, '2xl': 'auto' }
+const top = { base: -1, '2xl': 'auto' }
+export const TCYNavIndicator = () => {
+  const claimsQuery = useTCYClaims('all')
+  // Need to add logic to show this if the user has claims
+
+  if (claimsQuery.length === 0) return null
```

Comment:
blocking: this doesn't do what we think it does, see with an empty wallet

<img width="349" alt="image" src="https://github.com/user-attachments/assets/a5f73b08-6d30-4912-941e-b6502944da76" />

Probably want something like

 ```suggestion
  const hasClaims = useMemo(() => claimsQuery.some(query => query.data.length), [claimsQuery])
  if (!hasClaims) return null
```